dom-renderers/DOMRenderer.js

```diff
@@ -117,6 +117,29 @@ DOMRenderer.prototype.subscribe = function subscribe(type, preventDefault) {
 };
 
 /**
+ * Removes an EventListener of given type from the element on which it was
+ * registered.
+ *
+ * @method
+ *
+ * @param {String} type DOM event type (e.g. click, mouseover).
+ *
+ * @return {undefined} undefined
+ */
+DOMRenderer.prototype.unsubscribe = function unsubscribe(type) {
+    this._assertTargetLoaded();
+    this._target.subscribe[type] = false;
+
+    if (
+        this._target.listeners[type] || this._root.listeners[type]
+    ) {
+        var target = eventMap[type][1] ? this._root : this._target;
+        target.listeners[type] = null;
+        target.element.removeEventListener(type, this._boundTriggerEvent);
```

Comment:
Let's say I have a rather flat scene graph:

```javascript
var a = scene.addChild();
var b = scene.addChild();
```
Now I'm adding DOMElements to both:

```javascript
var aEl = new DOMElement(a);
var bEl = new DOMElement(b);
```
Now I'm adding an UIEvent to node `a` and `b`:

```javascript
a.addUIEvent('click');
b.addUIEvent('click');
```

Now I'm listening for click events in some way:

```javascript
a.addComponent({
    onReceive: function(type, ev) {
        console.log('a: ', type, ev);
    }
});
b.addComponent({
    onReceive: function(type, ev) {
        console.log('b: ', type, ev);
    }
});
```

Now when I click on the actual element in the DOM associated with `a` or `b`, the event will be console logged.

Now I'm removing the UIEvent from one node:

```javascript
a.removeUIEvent('click');
```

Now I'm clicking on b.

## Expected result

`console.log' with event

## Actual result

Nothing happens.

## Explanation

The click event handler has been attached to the root element. Removing the UIEvent from `a` resulted into the event listener being removed from the root element of the DOMRenderer.

## Solution

Never remove event listeners. Set `subscribe` to false instead.